mlir/include/mlir/Dialect/SPIRV/IR/SPIRVBase.td

```diff
@@ -3098,6 +3098,11 @@ def SPIRV_EM_SchedulerTargetFmaxMhzINTEL      : I32EnumAttrCase<"SchedulerTarget
     Capability<[SPIRV_C_FPGAKernelAttributesINTEL]>
   ];
 }
+def SPIRV_EM_StreamingInterfaceINTEL          : I32EnumAttrCase<"StreamingInterfaceINTEL", 6154> {
+  list<Availability> availability = [
+    Capability<[SPIRV_C_FPGAKernelAttributesINTEL]>
+  ];
+}
```

Comment:
I don't think this should be committed.
If you intend to extend SPIR-V dialect, please, commit this change to llvm.org project directly (or sycl-mlir branch).

Reply:
Okay.  I guess I am a bit unclear about exactly what goes where.  I can pull this file out of this PR and add it LLVM directly.  If I do so, are there any other files that should be updated too?

Reply:
Removed and added in [D140476](https://reviews.llvm.org/D140476)